metal/classifier.py

```diff
@@ -200,10 +200,9 @@ def _train_model(
         epoch_size = len(train_loader.dataset)
 
         # Move model to GPU
-        if self.config["use_cuda"]:
-            if self.config["verbose"]:
-                print("Using GPU...")
-            self.cuda()
+        if self.config["verbose"] and self.config["device"] != "cpu":
+            print("Using GPU...")
+        self.to(self.config["device"])
```

Comment:
I could be missing something, but I think this needs an indent to be inside the `if` statement?